Simulator/tft_simulator.py

```diff
@@ -255,12 +261,10 @@ def step(self, action):
         }
 
         self._clear_rewards()
-
         _non_trunc_agents = self.agents[:]
-        if self.taken_max_actions(agent):
+        if self.taken_max_actions(agent) or self.player_manager.player_states[agent].is_afk:
```

Comment:
If using truncations, this isn't needed. This is code that is run on every action so I am very picky about what gets put there.